openshift_tools/monitoring/pminfo.py

```diff
@@ -115,6 +118,36 @@ def connect(self):
         """ Establish a PMAPI context to archive, host or local, via args """
         self.context = pmapi.pmContext()
 
+def calculate_percent_cpu(start, end, interval):
+    ''' Calculate the percent of cpu utilization from sample1 and sample2.
+        Requires:
+          start
+          end
+          number of cpus
+          interval
+    '''
+    num_cpu = multiprocessing.cpu_count()
+    jiffy = os.sysconf(os.sysconf_names['SC_CLK_TCK'])
+    return ((end - start) * 100 / (float(jiffy) * interval)) / (num_cpu * 10)
+
+def get_sampled_data(metrics, interval):
```

Comment:
I think we probably want a count here as well.

Basically, I want to say "give me 10 samples, at 1 second intervals" and have this function take 10 seconds to give me a list of 10 values.

Basically, if I sample for 60 seconds, I don't just want 2 values over those 60 seconds, I want multiple, so I can then apply an average, median, stddev, etc to it.